docs/iris/src/userguide/cube_statistics.rst

```diff
@@ -301,7 +302,8 @@ do not cover a three month period (note: judged here as > 3*28 days):
 
 .. doctest:: aggregation
 
-    >>> spans_three_months = lambda t: (t.bound[1] - t.bound[0]) > 3*28*24.0
+    >>> tdelta_3mth = datetime.timedelta(hours=3*28*24.0)
```

Comment:
I'm OK with this here, but can we fix this up subsequently to use real months? Happy for you to delegate 😉